src/main/entity/Promotion.ts

```diff
@@ -14,6 +14,7 @@ import { PromotionType } from '../data/PromotionType';
 import { CuisineType } from '../data/CuisineType';
 import { SavedPromotion } from './SavedPromotion';
 import { Schedule } from './Schedule';
+import { VoteRecord, VoteState } from './VoteRecord';
```

Comment:
Just some linting errors, I usually run `yarn lintFix` before I push and it'll console output any warnings